ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Version.scala

```diff
@@ -80,6 +80,7 @@ object Version {
     *      * Add DamlSubmissionBatch message.
     *   4: * Remove application_id from DamlCommandDedupKey. Only submitter and commandId are used for deduplication.
     *      * Add deduplicatedUntil field to DamlCommandDedupValue to restrict the deduplication window.
+    *   5: * Add active_at to DamlContractKeyState to be able to check causal monotonicity for positive key lookups.
```

Comment:
Please refer to the definition of causal monotonicity or explicitly spell it out like, e.g.:
```suggestion
    *   5: * Add active_at to DamlContractKeyState to be able to check causal monotonicity for positive key lookups, i.e., whether all contracts that are an input to a transaction are active at a ledger effective time <= the ledger effective time as the transaction under validation.
```

Reply:
Fixed in 14a595f.